### ISSUE TITLE
jquery 3.2.1 load issue

### ISSUE THREAD

**Issue by shift-reality** [waynegm/imgViewer2#4]
Hi! In new version of jquery i got next error:

jQuery.Deferred exception: Cannot read property 'indexOf' of undefined TypeError: Cannot read property 'indexOf' of undefined
    at r.fn.init.r.fn.load (*jquery-3.2.1.min.js:4:18893)
    at HTMLImageElement.<anonymous> (*imgViewer2.min.js:1:3528)

can you help me? Thank you

**Comment by shift-reality** [comment]
probably issue in file https://github.com/waynegm/imgViewer2/blob/master/src/imgViewer2.js on line 161

**Comment by waynegm** [comment]
This is fixed in the Github master. The load() event trigger has been removed from jQuery 3 so I replaced it with a trigger("load") call. The change appears to work in older jQuery versions as well.

**Comment by shift-reality** [comment]
Work fine
Thank you

**Comment by shift-reality** [comment]
UPD: Work no correctly(
Plugin initialize only after window resize (zoom panel show only after resize window... mouse wheel same issue)

**Comment by shift-reality** [comment]
Sorry, This works fine now!
But i found another bug: this incompatible with jquery steps plugin
(plugin not initializing, init happens only after window resize... maybe event not subscribed) How i can fix it?
and image has position absolute breaks step page

**Comment by waynegm** [comment]
I've never used that plugin so I'm not sure how much I can help. Have you tried using the iframe  content mode?
